README.md

```diff
@@ -4,19 +4,20 @@ Configurable grayscale effect for KWin window manager and KDE desktop environmen
 
 # Installation
 
+Either install manual by coping files or use `kpackagetool6`. It may work with `kpackagetool5` but it is untested.
 
 ## Automated
 
 
-1. [Download](https://github.com/lliurex/kwin-grayscale-effect/archive/refs/heads/master.zip) or Git Pull
-2. Enter the repository directory, for example `cd kwin-grayscale-effect-master`
+1. [Download](https://github.com/lliurex/kwin-grayscale-effect/archive/refs/heads/plasma6.zip) or Git Pull
+2. Enter the repository directory, for example: `cd kwin-grayscale-effect-plasma6`
```

Comment:
Missing the step of unzipping the archive. It's trivial but can confuse new users.
The bash could be
```{.bash}
unzip kwin-grayscale-effect-plasma6.zip
cd kwin-grayscale-effect-plasma6
kpackagetool6 --type KWin/Effect --install kwin6_effect_grayscale
```

Reply:
Hi!
I've been busy with another tasks. Finally I've merged _your branch_ into master.
Many thanks for all!